schema/parameter.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$udmi_version": "1.5.1",
+    "title": "Parameter",
```

Comment:
"parameter" is a bit too generic here... I'd suggest maybe "measurement" -- https://english.stackexchange.com/questions/111291/what-is-a-name-for-a-unit-of-measure-and-value -- things like "dimension" or "quantity" also crop up, but usually in conjunction with something like a "dimensional quantity" which can get unwieldy. I just know "parameter" is bad.